laravel/app/Models/Config.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+/**
+ * App\Models\Config
+ *
+ * @property string $token
+ * @property string|null $value
+ * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Config whereToken($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|\App\Models\Config whereValue($value)
+ * @mixin \Eloquent
+ */
+class Config extends BaseModel
```

Comment:
This is an example of the usage of the 'Eloquent' ORM object that backs onto the database. 

Creating a new database row.

```
$item = new Config();
$item->token = 'something';
$item->value = 'somethingElse';
$item->save();
```

Fetching a database row

```
$item = Config::where('token', '=', 'something')->first();
echo $item->token;
```